kubernetes/docs/deploy-demo-graph.md

```diff
@@ -0,0 +1,44 @@
+How to deploy the Demo graph
```

Comment:
In the guides you refer to the 'demo' graph and the 'default' graph, but I think you are always referring to the same simpleGraph schema that the Gaffer Helm Chart uses that provides an empty graph. I think I'd interpret the 'demo' graph as being that provided by the Road Traffic Helm Chart as it provides a more realistic schema, with data.

Reply:
Okay. Maybe something like "empty" graph might be better?

Reply:
Sure